internal/venice-consumer/src/main/java/com/linkedin/venice/pubsub/api/PubSubTopic.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.venice.pubsub.api;
+
+public interface PubSubTopic {
+  /**
+   * @return the name of the topic
+   */
+  String getName();
+
+  /**
+   * @return the {@link PubSubTopicType} of the topic
+   */
+  PubSubTopicType getPubSubTopicType();
```

Comment:
Hrm.  Why do we think we need this?  A user of change capture will probably only subscribe to some 'view', and then based on the store config the events played back will be based on that view.  It's unclear to me how SR, RT or VT will be used from the interface perspective?  Maybe you have something in mind?

Reply:
The `PubSubTopic` interface is not only for change capture consumer client, it might be also leveraged by our ingestion logic in the future.

Reply:
But should we expose this to a client?  Do you have a concrete case where we'd need to know this?

Reply:
The PubSubTopic is not sth that a client will see, it is for our internal use.